crates/menmos-client/src/client.rs

```diff
@@ -360,7 +376,7 @@ impl Client {
     async fn push_internal<P: AsRef<Path>>(
         &self,
         path: P,
-        meta: Meta,
+        meta: &mut Meta,
```

Comment:
Again, this works but you _could_ change this to `mut Meta`, since push doesn't need to keep the original meta. This way you'd save yourself the `clone` call at L396